app/v2/notifications/post_notifications.py

```diff
@@ -406,6 +409,29 @@ def process_precompiled_letter_notifications(*, letter_data, api_key, template,
     return notification
 
 
+def validate_sender_id(template, reply_to_id):
```

Comment:
This is similar to `get_reply_to_text` just below, but it returns a UUID instead of the reply value (for emails) or sender for SMS

Reply:
A bit similar to `check_reply_to` in the `validators.py` file too if I get this right. Why do we hide the reply-to address behind a UUID? to hide the address itself when possible?

Reply:
This is mostly to follow what's done already on other API endpoints, when sending an email for example. You pass UUID for templates, reply to etc.

The UI is clear about the email address and its UUID. You can also update a reply to and keep the same email address.
<img width="1048" alt="Screen Shot 2021-06-22 at 08 22 06" src="https://user-images.githubusercontent.com/295709/122923766-06742580-d333-11eb-8c2a-53e90faa285d.png">